hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/event/remote/RemoteTenantAwareEventTest.java

```diff
@@ -69,10 +68,9 @@ public void testTargetAssignDistributionSetEvent() {
     private void assertTargetAssignDistributionSetEvent(final Action action,
             final TargetAssignDistributionSetEvent underTest) {
 
-        final Entry<String, Long> entry = new SimpleImmutableEntry(action.getTarget().getControllerId(),
-                action.getId());
-
-        assertThat(underTest.getActions()).containsExactly(entry);
```

Comment:
Shouldn't you better keep this assertion? Test only added a singe action to `TargetAssignDistributionSetEvent`. Now after serialisation/deserialisation you won't recognise if there is by accident more than one action in place

Reply:
The problem is, during marshalling and unmarshalling, new objects get created that have different objectids in memory of course. And the containsExactly() compares the java objectIds which in that case will not match.
Same for containsOnly(). This is why I had to remove it

Reply:
I see, totally fine. What about an additional `assertThat(underTest.getActions()).hasSize(1)` instead?